CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 * [Fix] Display a message instead of an error when `toml` isn't installed and `pyproject.toml` is found ([#825](https://github.com/ploomber/jupysql/issues/825))
 * [Fix] Fix argument parsing error on Windows when it contains quotations ([#425](https://github.com/ploomber/jupysql/issues/425))
 * [Fix] Fix error when a linebreak is included during nonidentifier validation process
+* [Feature] Improved messages when loading configurations from `pyproject.toml` file.
```

Comment:
we just made a release, please move this change to the top section in the changelog

Reply:
Fixed. Please check the changelog.md file directly, I'm not sure why I cant see the last commit